sdcm/utils/issues.py

```diff
@@ -0,0 +1,87 @@
+import re
+import sys
+import logging
+from functools import lru_cache
+from dataclasses import dataclass
+
+import github
+import github.Auth
+from github.GithubException import UnknownObjectException
+
+from sdcm.keystore import KeyStore
+from sdcm.sct_config import SCTConfiguration
+from sdcm.sct_events.base import Severity
+from sdcm.sct_events.system import TestFrameworkEvent
+
+scylla_issue_pattern = re.compile(
+    r"^\s*((((?P<user_id>[\w:-]+)/)?(?P<repo_id>[\w:-]+))?#)?(?P<issue_id>\d+)\s*$", re.IGNORECASE)
+
+scylla_issue_https_pattern = re.compile(
+    r"^https?:/.*?/(?P<user_id>[\w:-]+)/(?P<repo_id>[\w:-]+)/.*?/(?P<issue_id>\d+?)$", re.IGNORECASE)
+
+
+@dataclass
+class Issue:
+    user_id: str | None
+    repo_id: str | None
+    issue_id: int | None
+
+
+class SkipPerIssues:
+    def __init__(self, issues: list[str] | str, params: SCTConfiguration | dict):
+        self.params = params
+
+        github_access = KeyStore().get_json("github_access.json")
+        auth = github.Auth.Token(token=github_access["token"])
+        self.github = github.Github(auth=auth)
```

Comment:
need clear error message if this fails due e.g. token expiration. Also, should not fail the test in such case I believe.

Btw. hopefully this token has only read permissions.

Reply:
It uses the same token as dtest is using
 
And yes should be handled the same as issue not found